src/signals/IncidentMap/components/AddressLocation/AddressLocation.tsx

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+
+import { useCallback } from 'react'
+
+import { Paragraph } from '@amsterdam/asc-ui'
+import type { LatLngLiteral } from 'leaflet'
+
+import { formatAddress } from '../../../../shared/services/format-address'
+import type { PdokResponse } from '../../../../shared/services/map-location'
+import type { Address } from '../../../../types/address'
+import { StyledPDOKAutoSuggest } from '../../../incident/components/form/MapSelectors/Asset/Selector/DetailPanel/styled'
```

Comment:
I would import the PDOK component itself from components and style it if necessary. This way you create a dependancy with this DetailPanel.

Reply:
Done